observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,73 @@ These release notes track the development of the Network Observability Operator
+
+[id="network-observability-eBPF-performance-enhancements-1.6_{context}"]
+==== eBPF performance enhancements
+Experience improved performances of the eBPF agent, in terms of CPU and memory, with the following updates: 
+
+* The eBPF agent now uses TCX webhooks instead of TC.
+* The *NetObserv / Health* dashboard has a new section that shows eBPF metrics.
+** Based on the new eBPF metrics, an alert notifies you when the eBPF agent is dropping flows.
+* Loki storage demand decreases significantly now that duplicated flows are removed. Instead of having multiple, individual duplicated flows per network interface, there is one de-duplicated flow with a list of related network interfaces.
+
+[IMPORTANT]
+====
+With the duplicated flows update, the *Interface* and *Interface Direction* fields in the *Network Traffic* table are renamed to *Interfaces* and *Interface Directions*, so any bookmarked *Quick filter* queries using these fields need to be updated to `interfaces` and `ifdirections`. 
+====
+
+For more information, see xref:../../observability/network_observability/network-observability-operator-monitoring.adoc#network-observability-netobserv-dashboard-ebpf-agent-alerts_network_observability[Using the eBPF agent alert]. 
+and xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-quickfilternw-observe-network-traffic[Quick filters].
+
+
+[id="network-observability-ebpf-collection-filtering-1.6_{context}"]
+==== eBPF collection rule-based filtering
+You can use rule-based filtering to reduce the volume of created flows. When this option is enabled, the *Netobserv / Health* dashboard for eBPF agent statistics has the *Filtered flows rate* view. For more information, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-ebpf-flow-rule-filter_nw-observe-network-traffic[eBPF flow rule filter].
+
+[id="network-observability-technology-preview-1.6_{context}"]
+=== Technology Preview features
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. Note the following scope of support on the Red Hat Customer Portal for these features:
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+[id="network-observability-cli-1.6_{context}"]
+==== Network Observability CLI
+You can debug and troubleshoot network traffic issues without needing to install the Network Observability Operator by using the Network Observability CLI. Capture and visualize flow and packet data in real-time with no persistent storage requirement during the capture. For more information, see xref:../../../observability/network_observability/netobserv_cli/netobserv-cli-install.adoc#network-observability-netoberv-cli-about_netobserv-cli-install[Network Observability CLI] and link:https://access.redhat.com/errata/RHEA-2024:3869[Network Observability CLI 1.6.0]
```

Comment:
Neither link works

Reply:
Errata link not gonna work. CLI link - argh. Let me try something else.

Reply:
Got that CLI link to work 😓